recipes/ipysankeywidget/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "ipysankeywidget" %}
+{% set version = "0.2.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 1f3a1ab7481ad41c06f05e7524479109d6f5685371a85b800667253221c25fa4
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - ipywidgets >=7.0.0
+
+test:
+  imports:
+    - ipysankeywidget
+
+about:
+  home: http://github.com/ricklupton/ipysankeywidget
+  license: MIT
+  license_family: MIT
```

Comment:
Is the license file in the source tarball? If so can you add it here?
If not can you add it manually and open an issue upstream?